client/dashboard/me/mcp/setup/index.tsx

```diff
@@ -0,0 +1,437 @@
+		const baseConfig = {
+			command: 'npx',
+			args: [ '-y', '@automattic/mcp-wpcom-remote@latest' ],
+		};
+
+		switch ( client ) {
+			case 'claude':
+				return {
+					mcpServers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+			case 'vscode':
+				return {
+					servers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+			case 'cursor':
+				return {
+					mcpServers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+			case 'continue':
+				return {
+					mcpServers: [
+						{
+							name: serverName,
+							...baseConfig,
+						},
+					],
+				};
+			default:
+				return {
+					mcpServers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+		}
+	};
+
+	// Copy MCP configuration to clipboard
+	const copyToClipboard = async () => {
+		const configText = JSON.stringify( generateMcpConfig( selectedMcpClient ), null, 2 );
+
+		try {
+			await navigator.clipboard.writeText( configText );
+			setCopyStatus( 'success' );
+			setTimeout( () => setCopyStatus( 'idle' ), 2000 );
+		} catch ( err ) {
+			setCopyStatus( 'error' );
+			setTimeout( () => setCopyStatus( 'idle' ), 2000 );
+		}
+	};
+
+	// Helper function to get the appropriate icon based on copy status
+	const getCopyIcon = () => {
+		switch ( copyStatus ) {
+			case 'success':
+				return check;
+			case 'error':
+				return error;
+			default:
+				return copy;
+		}
+	};
+
+	// Handle error states only - allow loading to continue so reauth can show
+	if ( userSettingsError ) {
+		return null;
+	}
+
+	// Check if any account-level tools are enabled using the new nested structure
+	const hasEnabledTools = hasEnabledAccountTools( userSettings );
+
+	if ( ! hasEnabledTools ) {
+		return (
+			<PageLayout
+				size="small"
+				header={
+					<PageHeader
+						title={ __( 'MCP Client Setup' ) }
+						description={ __( 'Configure your MCP client to connect to WordPress.com.' ) }
+					/>
+				}
+			>
+				<ComponentViewTracker eventName="calypso_dashboard_mcp_setup_view" />
+				{ ! isLoadingUserSettings && (
+					<>
+						<SectionHeader label={ __( 'Setup Required' ) } />
+						<Card>
+							<CardBody>
+								<VStack spacing={ 4 }>
+									<Text as="p" size="medium">
+										{ __( 'No MCP tools are currently enabled for your account.' ) }
+									</Text>
+									<Text as="p" size="medium">
+										{ __(
+											'MCP tools define what actions and data your MCP client can access on your account. You need to enable at least one tool in the main MCP settings before configuring your client.'
+										) }
+									</Text>
+									<Link to="/me/mcp">
+										<Button variant="primary" style={ { alignSelf: 'flex-start' } }>
+											{ __( 'Go to MCP Settings' ) }
+										</Button>
+									</Link>
+								</VStack>
+							</CardBody>
+						</Card>
+					</>
+				) }
+			</PageLayout>
+		);
+	}
+
+	return (
+		<PageLayout size="small" header={ <PageHeader title={ __( 'MCP Client Setup' ) } /> }>
```

Comment:
Let's add `Breadcrumbs` to this page so the user can go back to previous page easily.

```
prefix={ <Breadcrumbs length={ 2 } /> }
```